google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Table.java

```diff
@@ -172,6 +176,10 @@ public Duration getChangeStreamRetention() {
     return changeStreamRetention;
   }
 
+  public boolean isProtected() {
```

Comment:
Add a comment:   /** Returns whether this table is deletion protected. */
and rename it to `isDeletionProtected` to be consistent with authorized view